manifests/repo.pp

```diff
@@ -33,17 +33,21 @@
 
   case $::osfamily {
     'Debian': {
-      if !defined(Class['apt']) {
-        class { 'apt': }
-      }
+      include ::apt
+
+      Class['apt::update'] -> Package<| |>
```

Comment:
Oops. This an old, old PR that doesn't reflect the current implementation.

If you have ideas for improving the code in `master` @baurmatt, please have at it! :)